MembraneRTC/Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -378,10 +378,12 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
      */
     public func setTrackBandwidth(trackId: String, bandwidth: BandwidthLimit) {
         guard let pc = connection else {
+            sdkLogger.error("setTrackBandwidth: Peer connection not yet established")
```

Comment:
In Elixir, logging function name can be turn on in metadata https://hexdocs.pm/logger/1.14/Logger.html#module-metadata. I found dedicated literals for iOS so you might consider using them https://stackoverflow.com/a/24049928/9620900